pywren/wren.py

```diff
@@ -64,7 +73,13 @@ def get_all_results(fs):
     call result on each one individually, and return those
```

Comment:
This is a typo from previous commit, can we get it fixed in this PR? "repeated" -> "completed"